src/chef-mover/src/mover_actor_keys_access_group_callback.erl

```diff
@@ -0,0 +1,158 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Tyler Cloke <tyler@chef.io>
+%% @copyright 2016 Chef Software, Inc.
+%%
+
+-module(mover_actor_keys_access_group_callback).
+
+-export([
+         migration_init/0,
+         migration_complete/0,
+         migration_type/0,
+         supervisor/0,
+         migration_start_worker_args/2,
+         migration_action/2,
+         next_object/0,
+         error_halts_migration/0,
+         reconfigure_object/2,
+         needs_account_dets/0
+        ]).
+
+-record(org, {name, id, authz_id}).
+-record(group, {name, id, authz_id}).
+-define(GLOBAL_PLACEHOLDER_ORG_ID, <<"00000000000000000000000000000000">>).
```

Comment:
I am pretty sure we have this available in an hrl we can pull in from somewhere?